support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -146,6 +149,11 @@ class CreateSubscriptionController(
 
 class LoggingCirceParser(controllerComponents: ControllerComponents) extends Circe {
 
+  override protected def onCirceError(e: io.circe.Error): Result = {
+    SafeLogger.warn(s"circe decode failure: $e")
+    super.onCirceError(e)
+  }
```

Comment:
this is all that's needed to get the circe failure logged, so I popped it in as an extra commit